includes/Admin/Settings.php

```diff
@@ -181,16 +181,16 @@ private function connect_to_enhanced_admin( $screen_id ) {
 			if ( ! empty( $_GET['tab'] ) ) {
 				switch ( $_GET['tab'] ) {
 					case Connection::ID:
-						$crumbs[] = __( 'Connection', 'facebook-for-woocommerce' );
+						$crumbs[] = __( 'Facebook for WooCommerce', 'facebook-for-woocommerce' );
 						break;
 					case Settings_Screens\Messenger::ID:
-						$crumbs[] = __( 'Messenger', 'facebook-for-woocommerce' );
+						$crumbs[] = __( 'Facebook for WooCommerce', 'facebook-for-woocommerce' );
 						break;
 					case Settings_Screens\Product_Sync::ID:
-						$crumbs[] = __( 'Product sync', 'facebook-for-woocommerce' );
+						$crumbs[] = __( 'Facebook for WooCommerce', 'facebook-for-woocommerce' );
 						break;
 					case Settings_Screens\Advertise::ID:
-						$crumbs[] = __( 'Advertise', 'facebook-for-woocommerce' );
+						$crumbs[] = __( 'Facebook for WooCommerce', 'facebook-for-woocommerce' );
```

Comment:
Any reason for default the crumb to a Single value? If this is changing then the switch case seem redundant

Reply:
This is as per a conversation we had with Matthias. We want to show Facebook for Woocommerce on top of the page instead of tab name. 
You're right. I'll remove the switch case